.pre-commit-config.yaml

```diff
@@ -36,5 +36,14 @@ repos:
           )
         args: ["-fallback-style=none", "-style=file", "-i"]
 
+  # TODO/REMINDER: add the Ruff vscode extension to the devcontainers
+  # Ruff, the Python auto-correcting linter/formatter written in Rust
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.8.1
```

Comment:
Latest is v0.8.2.
```suggestion
    rev: v0.8.2
```

Reply:
Done: https://github.com/NVIDIA/cccl/pull/3130/commits/8ac225484c96964e864644a587a0eb53979a7d01

(Currently in a scratch PR; once it's settled there I'll force push here.)